src/url.php

```diff
@@ -320,6 +321,36 @@ function qtranxf_get_url_info( string $url ): array {
     return $urlinfo;
 }
 
+/**
+ * Returns the base admin url of the WordPress backend name e.g. wp-admin
+ * If the $admin_base is empty it returns the standard WordPress backend name
+ *
+ * @author Sebastian Poetter https://github.com/poetter-sebastian
+ * @link https://github.com/qtranslate/qtranslate-xt/pull/1324 repo pull request
+ *
+ * @return string WordPress backend name
+ */
+function qtranxf_get_admin_base():string
+{
+    $admin_base = str_replace( [site_url(), '/'], '', admin_url() );
+    return strlen($admin_base) == 0? 'wp-admin' : $admin_base;
```

Comment:
- I'm not sure if there could be more complex cases where the custom base can contain slashes, it would be safer to change to `trim( str_replace( site_url(), '', admin_url() ), '/');`
-  No need to test here, we can just return the result of `trim` as in wp codebase `admin_url` is already built appending `wp-admin\` to `site_url`. Also the `wp-admin` could change upstream, though highly unlikely.